src/Business Foundation/App/NoSeriesCopilot/src/Register/NoSeriesCopilotInstall.Codeunit.al

```diff
@@ -0,0 +1,15 @@
+namespace Microsoft.Bank.Reconciliation;
+
+codeunit 290 "No. Series Copilot Install"
```

Comment:
You've got 2 of these codeunits 😉

Reply:
Yep, good that we have 2nd vision ;) Removed